app/RFCService.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use App\Types\Issue;
+use App\Types\Status;
+use DateTimeImmutable;
+use App\Helpers\GitHub;
+use App\Types\IssueType;
+use App\Types\GitHubUser;
+use Illuminate\Support\Arr;
+use Hyde\Markdown\Models\Markdown;
+
+class RFCService
+{
+    public function handle(): void
+    {
+        $issues = GitHub::request('get', 'issues', ['state' => 'all', 'labels' => 'RFC']);
```

Comment:
Could be merged to use the Search API https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-issues-and-pull-requests

Reply:
Yeah, we're gonna want this, as we can't filter PRs like we can Issues, so we will need to paginate through all PRs to filter out the ones we need.